float64 jointOne
float64 jointTwo
float64 jointThree
float64 jointFour